components/create-collective/sections/ConnectGithub.js

```diff
@@ -116,20 +123,21 @@ class ConnectGithub extends React.Component {
     const { token, intl, query } = this.props;
     const { repositories, loadingRepos, error } = this.state;
 
+    const FISCAL_SPONSOR_TERMS =
```

Comment:
I told you to do that and it's ok for now, but it's not nice at all.

Maybe there should be a special URL to do that. `http://opencollective.com/opensource/terms/redirect` Would fetch the URL and redirect the user.